src/components/timeseriesexplorer.js

```diff
@@ -20,6 +20,70 @@ const TimeSeries = lazy(() =>
   import('./timeseries' /* webpackChunkName: "TimeSeries" */)
 );
 
+function Chart({
+  isVisible,
+  stateCode,
+  timeseries,
+  dates,
+  chartType,
+  isUniform,
+  isLog,
+}) {
+  return (
+    isVisible && (
+      <Suspense fallback={<TimeseriesLoader />}>
+        <TimeSeries
+          {...{timeseries, dates, chartType, isUniform, isLog, stateCode}}
+        />
+      </Suspense>
+    )
+  );
+}
+
+function Pills(props) {
+  const {t} = useTranslation();
+  return (
+    <div className="pills">
+      {Object.values(TIMESERIES_OPTIONS).map((option) => (
+        <button
+          key={option}
+          type="button"
+          className={classnames({selected: props.timeseriesOption === option})}
+          onClick={() => props.setTimeseriesOption(option)}
+        >
+          {t(option)}
+        </button>
+      ))}
+    </div>
+  );
+}
+
+function Alert() {
+  const {t} = useTranslation();
+  return (
+    <div className="alert">
+      <IssueOpenedIcon size={24} />
+      <div className="alert-right">
+        {t('Tested chart is independent of uniform scaling')}
+      </div>
+    </div>
+  );
+}
+
+function Info() {
+  const {t} = useTranslation();
+  return (
+    <div className="alert info">
+      <IssueOpenedIcon size={24} />
+      <div className="text">
+        {t(
+          'There are no COVID-19 reports or data available at this point of time'
```

Comment:
I think the strings are supposed to be in [locales.json](https://github.com/covid19india/covid19india-react/blob/master/locales/all_locales.json) for localization. 
Not sure though 😕 .

Reply:
Thanks for pointing this out! I have updated both changes.
In the case of the string, I am not sure how it will reflect for every local language as I don't have the permission to update the data or I am not sure where it is kept.

Reply:
That is what I'm also unsure of. How they are managing the localization, but this change will make it ready for the future anyways. 💯

Reply:
Great ✌️